python/python/lance/ray/sink.py

```diff
@@ -129,8 +126,22 @@ def on_write_start(self):
 
     def on_write_complete(
         self,
-        write_results: List[List[Tuple[str, str]]],
+        write_results,
     ):
+        if not write_results:
+            import warnings
+
+            warnings.warn(
+                "write_result_blocks is empty.",
+                DeprecationWarning,
+            )
+            return "Empty list"
+
+        first_element = write_results[0]
+        if isinstance(first_element, (pa.Table, pd.DataFrame)):
+            write_results = [
+                result["write_result"].iloc[0]["result"] for result in write_results
+            ]
```

Comment:
Ah, does this work on older versions of Ray as well?  It's probably not a deal breaker if it doesn't (we can require a certain version of Ray) but I'm curious to know how we should document this.

Reply:
Yes, the changes here are compatible with older versions of Ray. So there is no need for a separate documentation. However, before the code I submitted to the Ray community is merged, only versions prior to Ray 2.38.0 can be used.